### PR TITLE
Enable tests on CI

### PR DESCRIPTION
**Pull request by ligurio**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,9 +35,28 @@ before_install:
     - eval "${MATRIX_EVAL}"
 
 before_script:
+    - DEBIAN_FRONTEND=noninteractive sudo -E apt update -y
+    - DEBIAN_FRONTEND=noninteractive sudo -E apt install -y libboost-all-dev
+    - sudo cp $TRAVIS_BUILD_DIR/test/data/travis.cnf /etc/mysql/conf.d/
+    - sudo service mysql restart
+    - mysql -u root -e "CREATE USER tarantool IDENTIFIED BY 'tarantool';"
```
Comment (LeonidVas) [inline .travis.yml]: I propose to move all this stuff (mysql...) to a script (something like in https://github.com/tarantool/mysql).

**Review by LeonidVas — CHANGES_REQUESTED** [review]
Hi! thank you for the patchset.

File: .travis.yml (hunk 1 above)
Comment (ligurio) [reply .travis.yml]: Thanks for review! I have moved MySQL setup to a separate script.

**Review by LeonidVas — APPROVED** [review]
LGTM.

**Pull request merged**